Study/task1.rb

```diff
@@ -0,0 +1,129 @@
+class Station
+
+  def initialize(name)
+    @name= name
+    @st = Array.new

```

Comment:
```suggestion
    @trains = []
```
имена лучше давать полные, а массив пустой инициализировать вот таким способом